Translations update from Weblate

weblate opened the pull request:
Translations update from [Weblate](https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/) for Kodi add-ons: music/plugin.audio.radio_de.



Current translation status:

![Weblate translation status](https://kodi.weblate.cloud/widgets/kodi-add-ons-music/-/plugin-audio-radio_de/horizontal-auto.svg)

gade01 reviewed (approved):
Everything looks good to me.

Corrections are done.
Translation memory has been used to translate some strings.

weblate committed:
Translated using Weblate (Asturian (Spain) (ast_es))

Currently translated at 9.0% (3 of 33 strings)

Translated using Weblate (Chinese (Taiwan) (zh_tw))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Chinese (China) (zh_cn))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Vietnamese (vi_vn))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Uzbek (uz_uz))

Currently translated at 9.0% (3 of 33 strings)

Translated using Weblate (Ukrainian (uk_ua))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Turkish (tr_tr))

Currently translated at 75.7% (25 of 33 strings)

Translated using Weblate (Thai (th_th))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Tamil (India) (ta_in))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Serbian (latin))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Serbian (sr_rs))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Albanian (sq_al))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Slovenian (sl_si))

Currently translated at 36.3% (12 of 33 strings)

Translated using Weblate (Slovak (sk_sk))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Russian (ru_ru))

Currently translated at 81.8% (27 of 33 strings)

Translated using Weblate (Romanian (ro_ro))

Currently translated at 27.2% (9 of 33 strings)

Translated using Weblate (Portuguese (Portugal) (pt_pt))

Currently translated at 81.8% (27 of 33 strings)

Translated using Weblate (Polish (pl_pl))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Dutch (nl_nl))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Norwegian Bokmål (nb_no))

Currently translated at 42.4% (14 of 33 strings)

Translated using Weblate (Burmese (my_mm))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Maltese (mt_mt))

Currently translated at 9.0% (3 of 33 strings)

Translated using Weblate (Malay (ms_my))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Mongolian (mn_mn))

Currently translated at 6.0% (2 of 33 strings)

Translated using Weblate (Malayalam (India) (ml_in))

Currently translated at 6.0% (2 of 33 strings)

Translated using Weblate (Macedonian (mk_mk))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Latvian (lv_lv))

Currently translated at 24.2% (8 of 33 strings)

Translated using Weblate (Lithuanian (lt_lt))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Korean (ko_kr))

Currently translated at 39.3% (13 of 33 strings)

Translated using Weblate (Japanese (ja_jp))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Italian (it_it))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Icelandic (is_is))

Currently translated at 24.2% (8 of 33 strings)

Translated using Weblate (Indonesian (id_id))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Armenian (hy_am))

Currently translated at 6.0% (2 of 33 strings)

Translated using Weblate (Hungarian (hu_hu))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Croatian (hr_hr))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Hindi (India) (hi_in))

Currently translated at 6.0% (2 of 33 strings)

Translated using Weblate (Hebrew (Israel) (he_il))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Galician (Spain) (gl_es))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (French (France) (fr_fr))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (French (Canada) (fr_ca))

Currently translated at 78.7% (26 of 33 strings)

Translated using Weblate (Faroese (fo_fo))

Currently translated at 12.1% (4 of 33 strings)

Translated using Weblate (Finnish (fi_fi))

Currently translated at 30.3% (10 of 33 strings)

Translated using Weblate (Persian (Iran) (fa_ir))

Currently translated at 9.0% (3 of 33 strings)

Translated using Weblate (Persian (Afghanistan) (fa_af))

Currently translated at 15.1% (5 of 33 strings)

Translated using Weblate (Basque (Spain) (eu_es))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Estonian (et_ee))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Spanish (Mexico) (es_mx))

Currently translated at 42.4% (14 of 33 strings)

Translated using Weblate (Spanish (Argentina) (es_ar))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Esperanto)

Currently translated at 9.0% (3 of 33 strings)

Translated using Weblate (Greek (el_gr))

Currently translated at 81.8% (27 of 33 strings)

Translated using Weblate (German (de_de))

Currently translated at 81.8% (27 of 33 strings)

Translated using Weblate (Welsh (United Kingdom) (cy_gb))

Currently translated at 21.2% (7 of 33 strings)

Translated using Weblate (Czech (cs_cz))

Currently translated at 18.1% (6 of 33 strings)

Translated using Weblate (Bosnian (Bosnia and Herzegovina) (bs_ba))

Currently translated at 6.0% (2 of 33 strings)

Translated using Weblate (Bulgarian (bg_bg))

Currently translated at 39.3% (13 of 33 strings)

Translated using Weblate (Azerbaijani (az_az))

Currently translated at 6.0% (2 of 33 strings)

Translated using Weblate (Afrikaans (South Africa) (af_za))

Currently translated at 18.1% (6 of 33 strings)

Added translation using Weblate (Asturian (Spain) (ast_es))

Translated using Weblate (Swedish (sv_se))

Currently translated at 100.0% (24 of 24 strings)

Translated using Weblate (Portuguese (Brazil) (pt_br))

Currently translated at 100.0% (24 of 24 strings)

Translated using Weblate (Danish (da_dk))

Currently translated at 100.0% (24 of 24 strings)

Translated using Weblate (Catalan (Spain) (ca_es))

Currently translated at 100.0% (24 of 24 strings)

Translated using Weblate (Belarusian (be_by))

Currently translated at 20.8% (5 of 24 strings)

Translated using Weblate (Arabic (Saudi Arabia) (ar_sa))

Currently translated at 100.0% (9 of 9 strings)

Translated using Weblate (Amharic (Ethiopia) (am_et))

Currently translated at 100.0% (4 of 4 strings)

Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Co-authored-by: Christian Gade <gade@kodi.tv>
Co-authored-by: Hosted Weblate <hosted@weblate.org>
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/af_za/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/am_et/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ar_sa/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ast_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/az_az/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/be_by/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/bg_bg/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/bs_ba/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ca_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/cs_cz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/cy_gb/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/da_dk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/de_de/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/el_gr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/eo/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/es_ar/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/es_mx/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/et_ee/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/eu_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/fa_af/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/fa_ir/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/fi_fi/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/fo_fo/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/fr_ca/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/fr_fr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/gl_es/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/he_il/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/hi_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/hr_hr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/hu_hu/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/hy_am/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/id_id/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/is_is/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/it_it/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ja_jp/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ko_kr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/lt_lt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/lv_lv/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/mk_mk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ml_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/mn_mn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ms_my/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/mt_mt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/my_mm/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/nb_no/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/nl_nl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/pl_pl/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/pt_br/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/pt_pt/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ro_ro/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ru_ru/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/sk_sk/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/sl_si/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/sq_al/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/sr_Latn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/sr_rs/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/sv_se/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/ta_in/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/th_th/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/tr_tr/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/uk_ua/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/uz_uz/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/vi_vn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/zh_cn/
Translate-URL: https://kodi.weblate.cloud/projects/kodi-add-ons-music/plugin-audio-radio_de/zh_tw/
Translation: Kodi add-ons: music/plugin.audio.radio_de
